retailer-oas/paths/timeslots/timeslots.yaml

```diff
@@ -39,22 +32,64 @@ get:
       example: 1
       schema:
         type: number
+    - name: service
+      in: query
+      default: "Pick&Delivery"
+      description: Which is the desired service to search in the Picking Locations Pool
+      type: string
+      enum: ["Pick&Delivery","Pick&Collect"]
+    - name: lat
+      in: query
+      schema:
+        type: number
+      format: ISO 6709:2022
+      example: 35.89421911
+      description: Latitude in decimal degrees
+    - name: lon
+      in: query
+      schema:
+        type: number
+      format: ISO 6709:2022
+      example: 139.94637467
+      description: Longitude in decimal degrees
     - name: postalCode
       in: query
+      description: The postal code of the shipping address
```

Comment:
question: only `postalCode` and `address` will be required?

Reply:
Postal code, addressline1, addressline2,  countryCode.
This is a proposal... the intention is for them to only use lon and lat... if that fails we need all these fields to fetch google maps API for the same info (lon, lat).